.github/workflows/techdocs-pypi.yml

```diff
@@ -0,0 +1,37 @@
+name: Master Build TechDocs PyPI Publish
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - '.github/workflows/techdocs-master.yml'
```

Comment:
Should point to this file

Reply:
Thanks @Rugvip 

@ellieseastream and @hooloovooo can you change this tomorrow please